tensorboard/__init__.py

```diff
@@ -19,14 +19,16 @@
 from __future__ import division
 from __future__ import print_function
 
+import importlib as _importlib
```

Comment:
nit: This exposes `tensorboard._importlib`; could resolve by lazily
importing `importlib` inside `program()` and `summary()` if you want.

Reply:
Fun fact: we're currently exposing the symbols `absolute_import`, `division`, and `print_function` as well :p

The main reason to use `importlib` was avoiding an inline import that requires a lint suppression internally, so at that point we can just directly import.